tests/python_client/testcases/test_collection.py

```diff
@@ -4496,3 +4496,125 @@ def test_create_collection_multiple_vectors_invalid_dim(self, get_invalid_dim):
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_drop_mmap_collection(self):
+        """
+        target: set mmap on collection
+        method: 1. set mmap on collection
+        2. drop collection
+        3. describe collection
+        expected: description information contains mmap
+        """
+        self._connect()
+        c_name = "coll_rand"
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=default_schema)
+        collection_w.set_properties({'mmap.enabled': True})
+        collection_w.drop()
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=default_schema)
+        pro = collection_w.describe().get("properties")
+        assert "mmap.enabled" not in pro.keys()
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_multiple_collections_enable_mmap(self):
+        """
+        target: enabling mmap for multiple collections in a single instance
+        method: enabling mmap for multiple collections in a single instance
+        expected: the collection description message for mmap is normal
+        """
+        self._connect()
+        c_name = "coll_1"
+        c_name2 = "coll_2"
+        c_name3 = "coll_3"
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=default_schema)
+        collection_w2, _ = self.collection_wrap.init_collection(c_name2, schema=default_schema)
+        collection_w3, _ = self.collection_wrap.init_collection(c_name3, schema=default_schema)
+        collection_w.set_properties({'mmap.enabled': True})
+        collection_w2.set_properties({'mmap.enabled': True})
+        pro = collection_w.describe().get("properties")
+        pro2 = collection_w2.describe().get("properties")
+        assert pro["mmap.enabled"] == 'True'
+        assert pro2["mmap.enabled"] == 'True'
+        collection_w3.set_properties({'mmap.enabled': True})
+        pro3 = collection_w3.describe().get("properties")
+        assert pro3["mmap.enabled"] == 'True'
+
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_flush_collection_mmap(self):
+        """
+        target: after flush, collection enables mmap
+        method: after flush, collection enables mmap
+        expected: the collection description message for mmap is normal
+        """
+        self._connect()
+        c_name = cf.gen_unique_str(prefix)
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=default_schema)
+        collection_w.insert(cf.gen_default_list_data())
+        collection_w.create_index(ct.default_float_vec_field_name, index_params=ct.default_flat_index,
+                                  index_name=ct.default_index_name)
+        collection_w.flush()
+        collection_w.set_properties({'mmap.enabled': True})
+        pro = collection_w.describe().get("properties")
+        assert pro["mmap.enabled"] == 'True'
+        collection_w.alter_index(ct.default_index_name, {'mmap.enabled': True})
```

Comment:
here lack of a case which set mmap false before and setting mmap as True by alter index